Makefile

```diff
@@ -34,16 +34,19 @@ clean: # Run `cargo clean`.
 
 .PHONY: lint
 lint: # Run `clippy` and `rustfmt`.
-	cargo +nightly fmt --all
+	cargo fmt --all
 	cargo clippy --all --all-targets --features "$(FEATURES)" --no-deps -- --deny warnings
 
 	# clippy for bls with supranational feature
 	cargo clippy --package ream-bls --all-targets --features "supranational" --no-deps -- --deny warnings
 
+	# cargo sort
+	cargo sort -g
+
 .PHONY: test
 test: # Run all tests.
 	cargo test --workspace -- --nocapture
 
 pr:
 	make lint && \
-	make test
+	make test
```

Comment:
![image](https://github.com/user-attachments/assets/5c48d2d3-2950-4a75-85f7-7933e180a02a)

this file needs a new line